src/disco/css/InstallButton.scss

```diff
@@ -115,9 +118,16 @@ $installStripeColor2: #00c42e;
       // impacting the animation speed.
       transform: scaleX(-1);
     }
+
     input + label {
       &::after {
         transform: translateX($switchHandleGap);
+
+        /* stylelint-disable max-nesting-depth */
```

Comment:
I'd rather there were more specific class names to remove some of this nesting rather than using a top level class and selecting on bare elements but that's definitely out of scope of making RTL work.

I don't think that was the wording we discussed in London but probably closed enough.